scripts/local/run.sh

```diff
@@ -7,40 +7,38 @@ set -e # Stop on first error
 source ./scripts/utils.sh
 source ./scripts/versions.sh
 
-LOCAL_RELAYER_IMAGE=
-RUN_STOP_FLAG="-c"
-HELP=
-while [ $# -gt 0 ]; do
-    case "$1" in
-        -l | --local-relayer-image) LOCAL_RELAYER_IMAGE=$2 ;;
-        -p | --pause) RUN_STOP_FLAG= ;;
-        -h | --help) HELP=true ;;
-    esac
-    shift
-done
-
-if [ "$HELP" = true ]; then
+function printHelp {
     echo "Usage: ./scripts/local/run.sh [OPTIONS]"
     echo "Run Teleporter."
     echo ""
     echo "Options:"
     echo "  -l, --local-relayer-image <tag>   Use a local AWM Relayer image instead of pulling from dockerhub"
     echo "  -p, --pause                       Pause the network on stop. Will attempt to restart the paused network on subsequent runs"
     echo "  -h, --help                        Print this help message"
-    exit 0
-fi
+}
 
 function cleanup {
     echo "Shutting down network before exiting..."
     ./scripts/local/run_stop.sh $RUN_STOP_FLAG
     echo "Network stopped"
 }
 
+LOCAL_RELAYER_IMAGE=
+RUN_STOP_FLAG="-c"
+while [ $# -gt 0 ]; do
+    case "$1" in
+        -l | --local-relayer-image) LOCAL_RELAYER_IMAGE=$2 ;;
+        -p | --pause) RUN_STOP_FLAG= ;;
+        -h | --help) printHelp && exit 0 ;;
+        *)  echo "Invalid option: $1" && printHelp && exit 1;;
+    esac
+    shift
+done
+
 # Set up the trap to catch the SIGINT signal (CTRL+C)
 # Note that the output of the cleanup function defined in run_setup.sh does not appear if CTRL+C is used to kill this script,
 # but the function does in fact run as expected.
-trap cleanup SIGTERM
-trap cleanup SIGINT
+trap cleanup SIGTERM SIGINT
```

Comment:
was the script hanging after CTRL+C? I thought that's what the trap was handling, and don't think ive had that issue previously

Reply:
No, it was hanging on line 64 if docker errored out. CTRL+C got it to exit, but you shouldn't have to in that situation.